resources/definitions/mingda_MagicianMax.def.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Could you change the definition file names to be lowercase with underscores, to be consistent with the rest?

Reply:
Thank you for your suggestion and I've changed it